pkgs/by-name/po/polymc-unwrapped/package.nix

```
@@ -0,0 +1,103 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  cmark,
+  extra-cmake-modules,
+  gamemode,
+  ghc_filesystem,
+  jdk8,
+  kdePackages,
+  ninja,
+  nix-update-script,
+  stripJavaArchivesHook,
+  tomlplusplus,
+  zlib,
+
+  msaClientID ? null,
+  gamemodeSupport ? stdenv.hostPlatform.isLinux,
+}:
+
+let
+  libnbtplusplus = fetchFromGitHub {
+    owner = "PolyMC";
+    repo = "libnbtplusplus";
+    rev = "2203af7eeb48c45398139b583615134efd8d407f";
+    hash = "sha256-TvVOjkUobYJD9itQYueELJX3wmecvEdCbJ0FinW2mL4=";
+  };
+in
+
+assert lib.assertMsg (
+  gamemodeSupport -> stdenv.hostPlatform.isLinux
+) "gamemodeSupport is only available on Linux.";
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "polymc-unwrapped";
+  version = "7.0";
+
+  src = fetchFromGitHub {
+    owner = "PolyMC";
+    repo = "PolyMC";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-d6Npu21iC5JUhWOM2DMn7gtzOj/Oo+ZoB36birHxzyA=";
+  };
+
+  postUnpack = ''
+    rm -rf source/libraries/libnbtplusplus
+    ln -s ${libnbtplusplus} source/libraries/libnbtplusplus
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    extra-cmake-modules
+    jdk8
+    stripJavaArchivesHook
+  ];
+
+  buildInputs = [
+    cmark
+    ghc_filesystem
+    kdePackages.qtbase
+    kdePackages.qtcharts
+    kdePackages.quazip
+    tomlplusplus
+    zlib
+  ] ++ lib.optional gamemodeSupport gamemode.dev;
```

```suggestion
  ] ++ lib.optional gamemodeSupport gamemode;
```
Shouldn't need to specify this